### PR TITLE
Fix key type for tnt and add tokenType

### PR DESCRIPTION
**Pull request by Perronef5**
### 📝 Description

- Fixed issue with TNT transactions sending the wrong key type.
- Added ability to support other token types.

### ❓ Context

- **Impacted projects**: `Helium HNT Wallet` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -39402,7 +39314,7 @@ packages:
     engines: {node: '>=8'}
 
   /path/0.12.7:
-    resolution: {integrity: sha1-1NwqUGxM4hl+tIHr/NWzbAFAsQ8=}
+    resolution: {integrity: sha512-aXXC6s+1w7otVF9UletFkFcDsJeO7lSZBPUQhtb5O0xJe8LtYhj/GxldoL09bBj9+ZmE2hNoHqQSFMN5fikh4Q==}
```
Comment (aelmilligy-ledger) [inline pnpm-lock.yaml:39317]: Do you know why this was updated ?

**Review by aelmilligy-ledger — CHANGES_REQUESTED** [review]
Hi @Perronef5, 

Thank you for your contribution. 
Could you please:
- Fix the conflicts
- Fix the Helium tests (check the CI if needed)

Thank you very much.

**Comment by Perronef5** [comment]
> Hi @Perronef5, 
> 
> 
> 
> Thank you for your contribution. 
> 
> Could you please:
> 
> - Fix the conflicts
> 
> - Fix the Helium tests (check the CI if needed)
> 
> 
> 
> Thank you very much.

Hi @aelmilligy-ledger 

I will make those changes by today 👍🏻

File: pnpm-lock.yaml (hunk 1 above)
Comment (Perronef5) [reply pnpm-lock.yaml:39317]: I just ran `pnpm install` Do you want me to revert that change?

**Review by gre — DISMISSED** [review]
the pnpm lock file appear broken. I would recommend to retake the pnpm-lock file from develop branch and `pnpm i` over it.

Screenshot 2022-08-29 at 13 47 57

let us know if you need help on this 🙏

**Comment by Perronef5** [comment]
> the pnpm lock file appear broken. I would recommend to retake the pnpm-lock file from develop branch and `pnpm i` over it.
> 
> Screenshot 2022-08-29 at 13 47 57
> 
> let us know if you need help on this 🙏

@gre I could really use your help here... The CI keep failing even when I didn't even touch those packages. I am doing:

```
git checkout develop pnpm-lock.yaml 
pnpm install
```

**Comment by valpinkman** [comment]
something weird has happened with the `compressjs` package. Here is your change
CleanShot 2022-09-09 at 10 30 41@2x
and here is what it looks like in `develop`

```yml
github.com/LedgerHQ/compressjs/d9e8e4d994923e0ea76a32b97289bcccfe71b82e:
    resolution: {tarball: https://codeload.github.com/LedgerHQ/compressjs/tar.gz/d9e8e4d994923e0ea76a32b97289bcccfe71b82e}
    name: '@ledgerhq/compressjs'
    version: 1.3.1
    hasBin: true
    dependencies:
      commander: 2.20.3
    dev: false
```

Not sure if it's one of the new dependencies you've added 
CleanShot 2022-09-09 at 10 33 04@2x
But definitely you're not supposed to have a `git+ssh` link inside the pnpm-lock

**Comment by Perronef5** [comment]
> something weird has happened with the `compressjs` package. Here is your change CleanShot 2022-09-09 at 10 30 41@2x and here is what it looks like in `develop`
> 
> ```yaml
> github.com/LedgerHQ/compressjs/d9e8e4d994923e0ea76a32b97289bcccfe71b82e:
>     resolution: {tarball: https://codeload.github.com/LedgerHQ/compressjs/tar.gz/d9e8e4d994923e0ea76a32b97289bcccfe71b82e}
>     name: '@ledgerhq/compressjs'
>     version: 1.3.1
>     hasBin: true
>     dependencies:
>       commander: 2.20.3
>     dev: false
> ```
> 
> Not sure if it's one of the new dependencies you've added CleanShot 2022-09-09 at 10 33 04@2x But definitely you're not supposed to have a `git+ssh` link inside the pnpm-lock
> 
> **EDIT:** Tried locally, copy the resolution from the `develop` branch instead of yours and try again (i tried to fix but i don't have the rights to push to your PR)

@valpinkman Thankyou for the help. I think I got it this time. May you please trigger the builds again 😀

**Review by haammar-ledger — APPROVED** [review]

**Pull request merged**